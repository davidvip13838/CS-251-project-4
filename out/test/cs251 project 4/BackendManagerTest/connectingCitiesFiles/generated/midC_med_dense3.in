500
34 12
17 38
82 36
69 74
0 81
48 12
37 6
65 24
57 36
22 78
47 56
79 72
64 27
94 83
3 94
80 96
77 17
95 95
8 32
68 18
74 84
62 73
6 93
92 32
15 5
72 81
78 41
25 64
62 78
76 64
6 6
73 97
11 73
62 66
54 86
56 45
92 94
33 40
81 75
82 1
94 97
69 60
65 81
67 62
80 69
24 42
32 42
53 39
24 56
0 40
44 83
98 44
75 79
100 98
34 82
16 48
13 77
69 82
15 18
41 55
1 37
79 81
15 91
80 72
81 22
23 67
32 12
86 64
42 6
30 80
72 68
44 11
12 44
76 37
78 61
30 76
75 99
5 6
56 41
37 31
41 71
31 39
11 32
45 69
78 67
47 21
30 71
9 7
81 3
5 32
70 80
28 100
69 90
40 57
72 16
27 36
31 5
71 42
42 13
30 47
95 4
0 42
1 95
28 66
30 9
9 28
95 96
27 35
13 90
65 65
21 45
92 61
25 79
96 99
98 7
12 6
10 96
2 9
55 58
33 0
5 70
52 68
94 44
75 81
32 68
35 88
0 0
59 53
0 47
20 56
42 63
71 2
27 71
43 51
85 57
8 24
76 95
81 48
27 21
96 86
58 82
36 100
99 46
93 29
49 90
26 92
80 33
83 56
34 39
48 48
95 38
80 54
49 98
30 89
22 47
87 76
9 0
77 59
83 15
84 87
61 36
83 67
56 87
71 100
78 23
21 84
76 65
40 19
64 77
48 74
21 62
42 72
35 54
45 25
21 55
71 81
7 88
45 39
69 29
22 67
16 51
78 59
6 63
65 16
6 17
91 56
33 73
73 100
78 46
97 81
6 20
86 85
76 33
5 34
26 69
91 99
8 89
22 22
66 21
99 61
16 49
7 98
4 74
63 2
99 96
16 5
82 11
46 84
97 20
2 27
75 36
8 64
34 48
83 69
64 12
43 55
90 70
31 81
52 16
4 95
70 27
62 75
22 68
84 65
4 85
73 93
39 95
46 1
16 26
31 53
61 17
77 56
5 100
12 78
42 27
28 71
50 79
15 31
94 90
6 50
56 72
90 3
1 7
82 69
6 14
75 58
29 73
89 59
3 61
35 58
95 19
85 91
56 75
49 67
34 14
1 3
53 28
76 61
99 68
37 86
88 68
23 54
87 86
50 83
58 85
82 64
15 13
10 33
71 8
10 47
73 44
9 8
79 98
21 70
19 49
64 47
4 31
99 92
88 100
77 84
87 90
92 93
20 46
25 59
74 3
48 88
97 84
42 2
98 80
17 65
40 79
63 98
2 4
67 83
100 64
59 83
90 21
97 24
91 100
34 63
90 78
24 92
59 57
32 76
55 1
65 79
40 63
89 94
5 52
25 72
26 97
4 77
42 58
85 17
90 43
75 93
31 82
36 50
100 13
47 40
3 50
51 52
51 35
97 64
64 64
31 40
59 35
59 24
50 11
33 36
85 38
73 39
91 9
50 18
91 42
3 81
21 65
7 91
78 85
78 47
9 70
25 24
65 62
23 53
97 92
9 89
90 54
67 9
73 76
67 47
72 29
45 87
19 85
18 6
1 46
91 16
3 46
64 82
51 45
62 35
79 11
61 79
33 82
41 22
84 15
99 89
80 63
11 14
100 77
43 40
20 22
8 98
6 81
7 56
27 45
53 100
56 59
6 80
61 62
81 83
51 26
82 2
44 4
40 66
84 88
97 62
24 32
19 75
80 39
26 99
13 33
14 26
59 84
75 80
3 41
73 84
61 42
51 82
27 39
34 41
16 32
26 81
56 85
30 8
82 57
45 33
34 88
15 16
52 17
50 60
58 69
49 58
93 18
20 27
57 99
54 80
15 87
41 89
63 66
8 26
76 55
10 72
99 33
37 100
89 16
43 94
84 0
15 86
18 69
25 16
37 39
96 0
55 97
64 55
50 88
60 69
36 21
38 42
95 44
70 33
20 11
39 26
56 1
36 30
93 98
25 68
40 83
36 41
71 43
25 96
49 59
35 99
69 87
0 96
84 27
99 67
98 17
88 29
25 21
76 58
89 99
66 69
62 41
72 50
1 88
41 88
72 38
68 89
37 74
42 37
25 49
63 18
26 89
93 74
90 41
92 87
90 96
71 66
5 94
10 80
6 10
96 98
73 64
11 39
64 86
92 3
94 68
14 25
90 64
99 78
66 85
70 57
36 72
48 26
97 57
60 19
36 32
64 34
76 87
64 96
