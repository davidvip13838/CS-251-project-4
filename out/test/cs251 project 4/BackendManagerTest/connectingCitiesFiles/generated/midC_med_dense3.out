1521.7154610781615